packages/documentation/copy/en/reference/Target Declaration.md

```diff
@@ -41,7 +41,7 @@ A target specification may have optional parameters, the names and values of whi
 - [**runtime-version**](#runtime-version): Specify which version of the runtime system to use.
 - [**rust-include**](#rust-include): (Rust only) A set of Rust modules in the generated project.
 - [**single-file-project**](#single-file-project): (Rust only) If true, enables [single-file project layout](#single-file-layout).
-- [**threading**](#threading): Whether to use multiple threads.
+- [**single-threaded**](#single-threaded): Whether to use a single-threaded or multi-threaded runtime.
```

Comment:
```suggestion
- [**single-threaded**](#single-threaded): Whether to use a single-threaded runtime (as opposed to a multi-threaded one).
```

Reply:
Thanks for the edit! I've made the other PR ready for review.